fs/move.py

```diff
@@ -4,11 +4,15 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+from os.path import commonpath
```

Comment:
Unfortunately this was only added in Python 3.5 https://docs.python.org/3/library/os.path.html#os.path.commonpath so this is causing the CI to fail on Python2.7 (see also #317 )

Reply:
Would it be ok if I copy this over to fs.path?

Reply:
(perhaps it would make a useful addition to https://docs.pyfilesystem.org/en/latest/reference/path.html ? Seems like it's be fairly to implement using `fs.path.recursepath`?)

Reply:
Urgh, 2.7 is kind of a bummer

Reply:
Ok so I'm planning to add `_pathcompat.py` and backport the function there